product-service/handlers/get-product-by-id/index.ts

```diff
@@ -0,0 +1,29 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import products from '../../mocks/products';
+
+export const handler: APIGatewayProxyHandler = async (event, _context) => {
+  try {
```

Comment:
I'd suggest moving duplicate logic (like error handling, response formatting) into separate lib/ functions and use them across the handlers.

It's a wish for the future, no need to implement it right now. It won't affect your score.